### PR TITLE
Implement Rock Rules

### PR DESCRIPTION
**Pull request by sarahabimay**
@Maikon

### CODE REVIEW

File: RockPaperScissors.iml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (jsuchy) [inline RockPaperScissors.iml:1]: This file should be ignored as well. Let us know if you need help with getting git to ignore the file after it's been committed like it has been here.

Also, make sure you ignore *.class. Might want to consider updating your ~/.gitignore_global to make sure compiled files are excluded, *.jar, *.iml, etc. Mine looks something like this:

```
# VIM swap files #
##################
*.swp

# Compiled source #
###################
*.com
*.class
*.dll
*.exe
*.o
*.so

# Packages #
############
# it's better to unpack these files and commit the raw source
# git has its own built in compression methods
*.7z
*.dmg
*.gz
*.iso
*.jar
*.rar
*.tar
*.zip

# Logs and databases #
######################
*.log
*.sqlite

# OS generated files #
######################
.DS_Store?
.DS_Store
ehthumbs.db
Icon?
Thumbs.db
```

File: src/SYMBOL.java [hunk 2]
```diff
@@ -0,0 +1,5 @@
+public enum SYMBOL {
```
Comment (jsuchy) [inline src/SYMBOL.java:1]: Enums names should follow same convention as classes and interfaces (Symbol)

File: test/RulesTest.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class RulesTest {
+
+    private RockRule rockRule;
+
+    @Before
+    public void setUp() throws Exception {
```
Comment (jsuchy) [inline test/RulesTest.java]: Please change your intellij template to remove the `throws Exception`. Checked exceptions are the worst, and we should avoid them as much as possible which means methods that advertise that they throw checked exceptions should be avoided.

File: test/RulesTest.java [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+        rockRule = new RockRule();
+    }
+
+    @Test
+    public void rockNotOneOfPlayersMove() {
+        assertEquals(null, rockRule.decideWinner(SYMBOL.PAPER, SYMBOL.PAPER));
+    }
+
+    @Test
+    public void rockBeatsScissors1() {
```
Comment (jsuchy) [inline test/RulesTest.java]: This test name smelled to me (1 and 2). Is the behaviour that we want to be getting the symbol that won? Or is it, given one person's choice and another person's choice, who won the Roshambo? If we change the answer to our question, these tests can probably be changed to not be smelly with the *1 and *2 suffix.

Comment (Maikon) [inline test/RulesTest.java]: I would say that `rockBeatsScissor1` & `rockBeatsScissor2` are testing the same logical assertion so in that case I would group them together in one test.

**Pull request merged**